packages/common/container-definitions/package.json

```diff
@@ -34,8 +34,8 @@
     "lint:fix": "npm run eslint:fix",
     "tsc": "tsc",
     "tsc:watch": "tsc --watch",
-    "typetests:gen": "flub generate typetests --generate --dir .",
-    "typetests:prepare": "flub generate typetests --prepare --dir . --pin"
+    "typetests:gen": "flub generate typetests --generate --dir . -s previousPatch",
+    "typetests:prepare": "flub generate typetests --prepare --dir . --pin -s previousPatch"
```

Comment:
The new flag is redundant given that this is a release branch, right? Per [this](https://github.com/microsoft/FluidFramework/blob/main/build-tools/packages/build-cli/docs/typetestDetails.md#branch-configuration). I don't mind having it there, just clarifying my understanding.

Reply:
You're right. I used this for testing; will remove.